codecards/src/components/AddCard.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const AddCard = ({ question, answer, handleInputChange, handleAddClick }) => {
+  return (
+    <div data-testid="addcard-component" className="addCardContainer">
+      <input className='questionText' type='text' name="questionInput" placeholder='Enter question here' value={question} onChange={handleInputChange}></input>
+      <input className='answerText' type='text' name="answerInput" placeholder='Enter answer here' value={answer} onChange={handleInputChange}></input>
+      <button className='addCard' onClick={handleAddClick}>Add to Deck</button>
```

Comment:
Rather than a button, this should be an input with `type=submit`. We could then make this onSubmit and that would allow users to submit by hitting enter. You can then style it the same as a button!